### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by mr-moon** [chobie/jira-api-restclient#41]
`composer require chobie/jira-api-restclient 2.0.*`

outputs:

```
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package chobie/jira-api-restclient could not be found in any
 version, there may be a typo in the package name.
```

**Comment by Pallieter** [comment]
I've ran a test install and get the same error. Composer checks are all okay.

**Comment by chrisforrence** [comment]
This is related to https://github.com/chobie/jira-api-restclient/issues/31

Basically, there are no available tags for 2.0. You would need to either pull dev-master or the 2.0.x-dev instead of 2.0.*.

* `composer require chobie/jira-api-restclient 2.0.x-dev`

**Comment by aik099** [comment]
Please use the `composer require chobie/jira-api-restclient 2.0@dev` command. The installation instructions should be updated as part of #33.
